plugins/kosync.koplugin/main.lua

```diff
@@ -23,9 +23,6 @@ end
 -- that was also used here as the interval for auto sync.
 -- DAUTO_SAVE_PAGING_COUNT has been removed, but let's allow
 -- this plugin to still pick it from defaults.persistent.lua.
---- @todo make this tunable via an added menu item below
-local SYNC_PAGING_COUNT = DAUTO_SAVE_PAGING_COUNT -- luacheck: ignore
```

Comment:
This comes along with editing defaults.lua. But there we comment it to signal it's no longer used, see some other values.

Reply:
Not exactly sure what you mean. Do you want me to get rid of that variable completely? I left it for the sake of compatibility (the comments above the highlighted lines). I'm fine removing `DAUTO_SAVE_PAGING_COUNT` completely, since in most cases it's `nil`

Reply:
I mean this looks like a PR similar to https://github.com/koreader/koreader/pull/5754. :-)

Reply:
I removed `DAUTO_SAVE_PAGING_COUNT`, should be good to go